scorecard.js

```diff
@@ -0,0 +1,19 @@
+const Frame = require('./frame');
+
```

Comment:
Great coding with splitting the functionalities into two separate classes and requiring one into the other. Excellent.